src/util/builtinOperators.ts

```diff
@@ -233,19 +233,29 @@ function globMatch(string: string, pattern: string): boolean {
 
 // generateGFunction is the factory method of the g(_, _) function.
 function generateGFunction(rm: rbac.RoleManager): any {
+  const memorized = new Map<string, boolean>();
```

Comment:
Will `memorized` be still available to access when `generateGFunction()` function exists? If yes, then merge.

Reply:
@hsluoyz `memorized` used to cache the results of `g()`, each call to `generateGFunction()` generates a new cache in enforcer.